content/articles/how-to-use-lambdas-to-manipulate-through-a-collection-of-objects/index.md

```diff
@@ -0,0 +1,372 @@
+
+### INTRODUCTION
+Java is majorly an object-oriented programming language. The importance of lambdas as a programmer is to
+make your code appear shorter, efficient, and above all functional. Everything in java is an object. 
+
+An object could be physical or imaginary, it just has to do with the way we see the thing about things in general. For example, a wallet is an object that has a method to store money, make money, add cards, get cards, etc.
+<!--more-->
+Example 2: a bucket is an object that has a method to store fluids either solid, liquid, or gas depending on what you want to store it with and get fluid.
+
+Now it gets interesting when you have a collection of wallets with different items in it and you want to access the one with the red card that has money more than #30,000 ('#' is naira in Nigeria currency), this is when lambdas come in because it hard you iterate through a collection without making mistakes and even if it works it's definitely not efficient.
+
+
+### Prerequisite
+- A basic understanding of the Java OOP(Object Oriented Programming) concept
+- Basic understanding of lambdas
+
+### Objectives
+
+1. To enable individuals to understand how lambdas work
+1. To enable individuals to understand how to sort
+1. To enable individuals to understand how to calculate the value of the different object
+1. To enable individuals to understand how to Group a collection of object 
+1. To enable individuals to understand how to check for a distinct object
+
+### Importance Of Lambdas
+Using lambdas is an effective way of writing your codes as some programmers argue that it might not be clear enough but the truth is, it is highly efficient and leaves room for no error in your code. 
+
+It abstracts all the pipe-line you have to create to find what you are looking for, let use our wallet object as a case study,  you will have to create different wallets based on color first before you create another group based on the outcome of the first group with money greater than '#30,000' and this could be time-consuming and also leads to a high time-complexity.
+
+Let code>>>
+First, we are going to create three(3)classes or objects.
+1. Card
+1. Wallet
+1. The Collection_Wallet as a driver class
+1. Input all of them inside into a folder call it Lambda_Collection
+
+```
```

Comment:
Add syntax highlighting for your language - https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Do this for all your code snippets.